### ISSUE TITLE
Quarry similar to landuse

### ISSUE THREAD

**Issue by math1985** [gravitystorm/openstreetmap-carto#2905]
Quarries look currently quite similar to residential on for example z12.

<img width="73" alt="screen shot 2017-10-24 at 01 50 54" src="https://user-images.githubusercontent.com/5251909/31918597-d209a890-b85d-11e7-9b2d-b25dff9ef3f6.png">

 Would using construction color be a good idea?

**Comment by kocio-pl** [comment]
I don't really know, it needs testing.

**Comment by Tomasz-W** [comment]
@matthijsmelissen Can you provide more examples? The one you have uploaded doesn't say too much.

**Comment by Adamant36** [comment]
@Tomasz-W, here is some more examples. 
![quarry 1](https://user-images.githubusercontent.com/30259065/44945302-0928b900-ad9b-11e8-823c-b3cefdd992ab.png)
![quarry 2](https://user-images.githubusercontent.com/30259065/44945303-0c23a980-ad9b-11e8-877c-9ce60a1a5e35.png)
![quarry 3](https://user-images.githubusercontent.com/30259065/44945305-0fb73080-ad9b-11e8-8a54-f7aad1d32005.png)

Although it doesn't look that bad, it might make sense to use a different color like construction. I can probably test it if anyone wants me to.

**Comment by kocio-pl** [comment]
Is it really z12?

**Comment by kocio-pl** [comment]
BTW: If you have any ideas to test, just go on and try them - you never know which will work or just inspire someone. :smile:

**Comment by Tomasz-W** [comment]
Yeah, the color of quarry area completly vanishes at z12 and higher zooms, and it becomes the same as residential area.

https://www.openstreetmap.org/way/244344780#map=12/41.9666/12.7465

**Comment by Adamant36** [comment]
My bad @kocio-pl. Here's some examples at z12. 
![quarry z12 2](https://user-images.githubusercontent.com/30259065/44945581-56a82480-ada1-11e8-931e-e6fd239130c1.png)
![quarry z12 1](https://user-images.githubusercontent.com/30259065/44945582-5871e800-ada1-11e8-8717-908603ac637e.png) 
I guess its between making it a different shade of gray at z12 or making it a different color altogether.

**Comment by meased** [comment]
What about rendering quarry identical to `landuse=industrial` but with the pickaxe pattern?

**Comment by polarbearing** [comment]
In high zoom quarry is well distinguishable from residential. Only the z12 issue should be addressed.

**Comment by dieterdreist** [comment]
sent from a phone

> On 1. Sep 2018, at 18:36, meased <notifications@github.com> wrote:
> 
> What about rendering quarry identical to landuse=industrial but with the pickaxe pattern?


I’m not in favor, these should be quite different because of the different nature, the current colors are working fine wrt being distinguishable

**Comment by Adamant36** [comment]
@dieterdreist, not that I necessarily disagree with you or @polarbearing, but what do you consider an "industrial" area versus whatever a rock quarry is landuse wise? Would it require factories or something? I ask because as it is things like water treatment plants are the same color as industrial landuse and I would put them in the same category as a rock quarry. Although I'm not that knowledgeable about those types of things.

**Comment by jeisenbe** [comment]
This issue happens because most types developed land (residential, industrial, commercial and retail) are unified to a slightly darker color at z12, and then all landuse colors are lightened, including quarry. This means that the quarry color and developed-landuse color at z12 are very similar. At z11 and z10 the developed land color stays darker while the quarry color and all other landcover colors get lighter, so the problem becomes less obvious. 

So this could be fixed by moving the landcover fading to lower zoom levels or removing it entirely, as we've been discussing. Changing the quarry color (eg unifying it with brownfield, construction, landfill) is possible, but not necessary to solve the issue if the fading is changed.

**Comment by Adamant36** [comment]
I assume you mean removing fadding for quaries only and not everything? Otherwise, that's a little excessive just to fix one issue. 

Changing quarry to be like landfill etc would be a good option. The grey color is a little odd anyway. Plus it would simplify things. Its just a matter of how the pattern would look with the landfill color I guess.

**Comment by dieterdreist** [comment]
sent from a phone

> On 29. Jan 2019, at 03:44, Adamant36 <notifications@github.com> wrote:
> 
> I assume you mean removing fadding for quaries only and not everything? Otherwise, that's a little excessive just to fix one issue.


there has been more criticism on landuse fading than just this one issue

**Comment by Adamant36** [comment]
@dieterdreist, your correct. I hadn't really thought about it, but there has been other criticism of fading. Personally I would like to see it modified, but it should be discussed in its own issue (I looked and couldn't find one related to it, but that might just be me. Maybe #3513?). There's a lot going on with land cover color changes lately.

**Comment by jragusa** [comment]
A quarry is much more closer to a landfill/construction site than a residential area. In some cases, the hole left at the end of the operation is filled with inert materials such as construction wastes or garbages.

I just gimped a quarry site with construction site colour (left) and with current quarry colour (right)
https://www.openstreetmap.org/#map=15/46.1305/6.4964
![quarry_test](https://user-images.githubusercontent.com/9897203/75535004-5aa81780-5a15-11ea-9b2d-6dd22905d7a5.png)
